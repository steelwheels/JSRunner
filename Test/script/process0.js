/* process0.js */
let proc0  = system("/bin/ls", stdin, stdout, stderr) ;

if(false){
	let ecode0 = proc0.waitUntilExit() ;
	console.log("ecode = " + ecode0) ;
} else {
	let ecode0 = _waitUntilExitAll([proc0]) ;
	console.log("ecode = " + ecode0) ;
}

