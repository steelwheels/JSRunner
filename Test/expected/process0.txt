Info.plist
JSTools.entitlements
JSTools.xcodeproj
ecode = 0
